arcgis-ios-sdk-samples/Features/Add delete related features/RelatedFeaturesViewController.swift

```diff
@@ -130,15 +130,19 @@ class RelatedFeaturesViewController: UITableViewController {
         // show progress hud
         SVProgressHUD.show(withStatus: "Applying edits")
         
-        relatedTable.applyEdits { [weak self] (_, error) in
+        relatedTable.applyEdits { [weak self] (results, error) in
             SVProgressHUD.dismiss()
             
-            if let error = error {
-                // show error
+            if let firstResult = results?.first {
```

Comment:
I'm not pretty sure here as the original sample isn't quite well structured.

The sample's UI only allows applying 1 edit each time, so it is safe to assume there is only 1 result. But in real case there could be many local edits and all of them get applied at once. Looking forward to suggestions.

Reply:
Hm, if you want to make changes to the sample to support multiple edits at once, you could try only calling the query method once "Done" is tapped.